dll/win32/getuname/getuname.c

```diff
@@ -18,14 +18,13 @@ GetUName(IN WORD wCharCode,
          OUT LPWSTR lpBuf)
 {
     WCHAR szDescription[256];
-    int res = LoadStringW(hInstance, wCharCode, szDescription, 256);
-    if (res != 0)
+    if (LoadStringW(hInstance, wCharCode, szDescription, 256))
```

Comment:
Not an improvement.

Reply:
Isn't? LoadString returns 0(false) if no string got found for a specific code or if it pulls an empty string, and and a nonzero value(true) if there is an appropriate string in the resource table.

Reply:
No, you are right.
Should i revert the change?

Reply:
Yes, please. It seems unnecessary.
Thanks.

Reply:
Done